src/browser_action/vitals.js

```diff
@@ -89,8 +96,9 @@
     chrome.storage.sync.get({
       enableOverlay: false,
       debug: false,
+      userTiming: false,
```

Comment:
Could you remove the non-INP changes from this PR?

Reply:
Done.  Since I had rebased I created a patch to remove the User Timing changes.

I do know that folks are finding it useful, so I created a separate branch which I'll need to rebase after this is landed.

Reply:
https://github.com/mmocny/web-vitals-extension/tree/inp_plus_ut